Mobile/iOS/Release 3.4.7/567-touch_id.md

```diff
@@ -0,0 +1,24 @@
+###  Touch ID 
+
+#### Pr: https://github.com/owncloud/ios/pull/567 
+
+Devices: iPhone6 v9.0.2, iPad Air v8.4
+
+Server: 8.2.2
+
+---
+
+ 
+| Test Case | Steps | Expected Result | Result | Related Comment
+:------: | :------------- | :------------- | :-------------- | :-----: | :------
+| Without Having finger touch ID enable on the device | 1.- Enable pincode within the app  | Touch ID option is not shown |
```

Comment:
Should be 'passcode' instead of 'pincode', here and in all the later appearances